apps/messagegui/app.js

```diff
@@ -361,12 +361,12 @@ function showMessage(msgid, persist) {
   active = "message";
   // Normal text message display
   let src=msg.src||/*LANG*/"Message", srcFont = fontSmall;
-  let title=msg.title, titleFont = fontLarge, lines;
+  let title=msg.title||"", titleFont = fontLarge, lines=[];
   let body=msg.body, bodyFont = fontLarge;
   // If no body, use the title text instead...
-  if (body===undefined) {
+  if (!body) {
     body = title;
-    title = undefined;
+    title = "";
   }
   if (g.setFont(srcFont).stringWidth(src) > g.getWidth()-52)
     srcFont = "4x6";
```

Comment:
Thanks, looks good! To avoid a future regression, perhaps we comment about the relation here?

```suggestion
    srcFont = "4x6";
    // one of `title` or `body` must be truthy at this point (#3969)
```

Reply:
Thanks!